backend/de.metas.business/src/main/java/de/metas/banking/api/impl/BPBankAccountDAO.java

```diff
@@ -111,6 +111,20 @@ public Optional<BankAccount> getDefaultBankAccount(@NonNull final BPartnerId bPa
 				.map(BPBankAccountDAO::toBankAccount);
 	}
 
+	@Override
+	public Optional<BankAccount> getDefaultESRBankAccount(@NonNull final BPartnerId bpartnerId)
```

Comment:
i think this should be cached. e.g.  idea: add an additional cache `bankAccountsByBPartnerId<BPartnerId, List<BankAccount>`